2018-12-17 18:16:40,583 ERROR: Exception on /user/test1 [GET] [in c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py:1560]
Traceback (most recent call last):
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Marlon Pimentel\microblog\app\main\routes.py", line 76, in user
    next_url=next_url, prev_url=prev_url)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\flask\templating.py", line 133, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Marlon Pimentel\microblog\app\templates\user.html", line 33, in template
    <span aria-hidden="true">&larr;</span> {{ _('Newer posts') }
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 875, in subparse
    add_data(self.parse_tuple(with_condexpr=True))
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 620, in parse_tuple
    args.append(parse())
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 432, in parse_expression
    return self.parse_condexpr()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 437, in parse_condexpr
    expr1 = self.parse_or()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 450, in parse_or
    left = self.parse_and()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 459, in parse_and
    left = self.parse_not()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 470, in parse_not
    return self.parse_compare()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 474, in parse_compare
    expr = self.parse_math1()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 496, in parse_math1
    left = self.parse_concat()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 507, in parse_concat
    args = [self.parse_math2()]
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 517, in parse_math2
    left = self.parse_pow()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 528, in parse_pow
    left = self.parse_unary()
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 547, in parse_unary
    node = self.parse_postfix(node)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 676, in parse_postfix
    node = self.parse_call(node)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\parser.py", line 793, in parse_call
    self.stream.expect('rparen')
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\lexer.py", line 388, in expect
    next(self)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\lexer.py", line 359, in __next__
    self.current = next(self._iter)
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\lexer.py", line 562, in wrap
    for lineno, token, value in stream:
  File "c:\users\marlon pimentel\microblog\venv\lib\site-packages\jinja2\lexer.py", line 683, in tokeniter
    filename)
jinja2.exceptions.TemplateSyntaxError: unexpected '}'
2018-12-17 18:18:49,006 INFO: Microblog startup [in C:\Users\Marlon Pimentel\microblog\app\__init__.py:75]
